Libraries/LibWeb/Crypto/CryptoAlgorithms.cpp

```diff
@@ -3145,16 +3143,16 @@ WebIDL::ExceptionOr<GC::Ref<JS::Object>> ECDSA::export_key(Bindings::KeyFormat f
                 },
                 [&](::Crypto::PK::ECPrivateKey<> const& private_key) -> ErrorOr<void> {
                     size_t coord_size;
-                    Variant<Empty, ::Crypto::Curves::SECP256r1, ::Crypto::Curves::SECP384r1> curve;
+                    Variant<Empty, ::Crypto::Curves::SECP256r1, ::Crypto::Curves::SECP384r1, ::Crypto::Curves::SECP521r1> curve;
                     if (algorithm.named_curve() == "P-256"sv) {
                         curve = ::Crypto::Curves::SECP256r1 {};
-                        coord_size = 256 / 8;
+                        coord_size = 32;
```

Comment:
Why add magic to the magic numbers?

Reply:
I've put them back to what they were before and used `ceil_div` for P521.